deploy/08_deployment.yaml

```diff
@@ -34,12 +34,12 @@ spec:
           - "operator"
           imagePullPolicy: Always
           env:
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
             - name: OPERATOR_NAME
               value: "jobset-operator"
+            - name: OPERATOR_IMAGE
```

Comment:
Why do we need OPERATOR_IMAGE?. We are operator?

Reply:
We do not, but I kept it as we usually reference that in our operators.

Reply:
Make sense and let's continue keeping it.